### PR TITLE
tlf_journal_test: fix race in TestTLFJournal/*WhileFlushing

### PR DESCRIPTION
**Pull request by strib**
In rare cases, `testTLFJournalPauseBlocksAndConvertBranch` can return
before the journal is converted to a branch.  In that case wait
explicitly for the branch change before moving onto the next part of
the test.

cc: @songgao 

Issue: KBFS-1985

### CODE REVIEW

File: libkbfs/tlf_journal_test.go [hunk 1]
```diff
@@ -1148,6 +1159,13 @@ func testTLFJournalPauseBlocksAndConvertBranch(t *testing.T,
 		markers++
 	}
 
+	// Wait for the local squash branch to appear.
+	select {
+	case <-branchCh:
+	case <-ctx.Done():
```
Comment (songgao) [inline libkbfs/tlf_journal_test.go]: I can't find a deadline or timeout on this ctx. If we rely on `-test.timeout`, maybe this isn't necessary?

Comment (strib) [inline libkbfs/tlf_journal_test.go]: Look in `setupTLFJournalTest()`.

Comment (songgao) [inline libkbfs/tlf_journal_test.go]: ah I see. Thanks!

**Comment by strib** [comment]
circleci actually succeeded.

**Pull request merged**